miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/MiniApp.kt

```diff
@@ -31,6 +36,8 @@ abstract class MiniApp internal constructor() {
         versionId: String
     ): MiniAppDisplay
 
+    protected abstract fun updateConfiguration(settings: MiniAppSdkConfig)
```

Comment:
Let's make this `internal`.
And, place it after all the public API methods in the file (helps with the reading and lookup). What do you think?